bitcoin/src/address/mod.rs

```diff
@@ -51,11 +51,11 @@ use crate::network::{Network, NetworkKind};
 use crate::prelude::*;
 use crate::taproot::TapNodeHash;
 
-use self::error::P2shError;
+use self::error::{NetworkError, P2shError};
 #[rustfmt::skip]                // Keep public re-exports separate.
 #[doc(inline)]
 pub use self::{
-    error::{Error, ParseError, UnknownAddressTypeError, UnknownHrpError, FromScriptError},
+    error::{ParseError, UnknownAddressTypeError, UnknownHrpError, FromScriptError},
```

Comment:
@Kixunil can you weigh in here, I personally think we should re-export public error types (excluding hider errors) from the modules where they are used. But we (you) just removed this in `hex`.